lib/presentation/mixins/contacts_view_controller_mixin.dart

```diff
@@ -316,8 +316,5 @@ mixin class ContactsViewControllerMixin {
     textEditingController.clear();
     searchFocusNode.dispose();
     textEditingController.dispose();
-    presentationContactNotifier.dispose();
```

Comment:
can you explain why remove those lines ?

Reply:
I reverted it and write differently that is clearer and easier to understand.